apps/projects/contracts/Projects.sol

```diff
@@ -781,40 +754,68 @@ contract Projects is AragonApp, DepositableStorage {
 // Internal functions
 ///////////////////////
 
-    /**
-     * @dev checks the hashed contract code to ensure it matches the provided hash
-     */
-    function _isBountiesContractValid(address _bountyRegistry) internal view returns(bool) {
-        if (_bountyRegistry == address(0)) {
-            return false;
-        }
-        if (_bountyRegistry == address(bountiesRegistry)) {
-            return true;
-        }
-        uint256 size;
-        // solium-disable-next-line security/no-inline-assembly
-        assembly { size := extcodesize(_bountyRegistry) }
-        if (size != 23406) {
-            return false;
+    function _setRepo(
+        bytes32 _repoId,
+        bool _decoupled,
+        string _repoData
+    ) internal
+    {
+        repos[_repoId].repoData = _repoData;
+        if (!isRepoAdded(_repoId)) {
+            repoIndex[repoIndexLength] = _repoId;
+            repos[_repoId].index = repoIndexLength++;
+            repos[_repoId].decoupled = _decoupled;
+            emit RepoAdded(_repoId, repos[_repoId].index, _decoupled);
+        } else {
+            emit RepoUpdated(_repoId, repos[_repoId].index, _repoData);
         }
-        uint256 segments = 4;
-        uint256 segmentLength = size / segments;
-        bytes memory registryCode = new bytes(segmentLength);
-        bytes32[4] memory validRegistryHashes = [
-            bytes32(0x9904de0ff2a8144b30f80f0de9184731b7c39116b1f021bad12dcbb740f8371d),
-            bytes32(0xd2319fa5b8b5614a3634c84ff340d27fa6e5921162e44bc2256f379ad86608f3),
-            bytes32(0x0fd4c8d32b2c21b41989666a6d19f7a5f4987ae6d915dd96698de62db8a79643),
-            bytes32(0x6af9efdc22f9352086c68a7b5c270db4f0fdc2b5ab18984a2d17b92ae327e144)
-        ];
-        for (uint256 i = 0; i < segments; i++) {
-            // solium-disable-next-line security/no-inline-assembly
-            assembly{ extcodecopy(_bountyRegistry,add(0x20,registryCode),div(mul(i,segmentLength),segments),segmentLength) }
-            if (validRegistryHashes[i] != keccak256(registryCode)) {
-                return false;
+    }
+
+    function _addBounties(
+        bytes32[] _repoIds,
+        uint256[] _issueNumbers,
+        uint256[] _bountySizes,
+        uint256[] _deadlines,
+        uint256[] _tokenTypes,
+        address[] _tokenContracts,
+        string _ipfsAddresses,
+        string _description,
+        bool _open
+    ) internal
+    {
+        // ensure the transvalue passed equals transaction value
+        //checkTransValueEqualsMessageValue(msg.value, _bountySizes,_tokenBounties);
```

Comment:
Is this comment relevant anymore or just dead code? We should remove to avoid confusion